site/frontend/src/pages/compare/benchmarks/benchmarks.vue

```diff
@@ -0,0 +1,72 @@
+<script setup lang="tsx">
+import {computed, h} from "vue";
+import TestCasesTable from "./test-cases-table.vue";
+import {TestCase} from "../data";
+import {CompareResponse, DataFilter} from "../types";
+
+export interface BenchmarkProps {
+  data: CompareResponse;
+  testCases: TestCase[];
+  allTestCases: TestCase[];
+  filter: DataFilter;
+  stat: string;
+}
+
+const props = defineProps<BenchmarkProps>();
+
+function Section({title, link, linkUp}: {title: string, link: string, linkUp: boolean}) {
+  return (
+    <div class="category-title">
+      {title} benchmarks
+      <span title={`To ${link} benchmarks`}>&nbsp;
+        <a href={`#${link}-benchmarks`}>{linkUp ? "⮭" : "⮯"}</a>
```

Comment:
This doesn't seem to render properly in the diff?

<img width="388" alt="Screenshot 2023-05-19 at 14 49 27" src="https://github.com/rust-lang/rustc-perf/assets/1327285/716d5a64-2c5b-4728-a208-0c1a9f6c00bb">

Reply:
It's fine in my source editor, probably just some visual bug.